module/item/sheets/WitcherArmorSheet.js

```diff
@@ -1,5 +1,6 @@
 import WitcherConfigurationSheet from './configurations/WitcherConfigurationSheet.js';
 import WitcherItemSheet from './WitcherItemSheet.js';
+import { associatedDiagramMixin } from '../../actor/sheets/mixins/associatedDiagramMixin.js';
```

Comment:
item sheets should use their own mixins. Here you access the mixins of the actor